packages/logging/src/loggers/base.logger.ts

```diff
@@ -2,12 +2,40 @@ import {LogModel} from "../models/log.model";
 import {SeverityEnum} from "../enums/severity.enum";
 import {Utils} from "../utils/utils";
 import {CommonModuleKeyname} from "@pristine-ts/common";
-import {Readable} from "stream";
 import {OutputModeEnum} from "../enums/output-mode.enum";
 
+/**
+ * The BaseLogger is the base abstract class that all loggers should extend.
```

Comment:
Not all loggers, it's mostly our own internal logs. This could give the impression that you must extend this but it's not mandatory. You could, but you could also not want all the functionalities there.